recipes/brotli/all/patches/0002-target-props.patch

```diff
@@ -17,22 +17,22 @@
  foreach(lib IN LISTS BROTLI_SHARED_LIBS BROTLI_STATIC_LIBS)
    target_link_libraries(${lib} ${LIBM_LIBRARY})
    set_property(TARGET ${lib} APPEND PROPERTY INCLUDE_DIRECTORIES ${BROTLI_INCLUDE_DIRS})
-@@ -190,7 +200,6 @@ foreach(lib IN LISTS BROTLI_SHARED_LIBS BROTLI_STATIC_LIBS)
+@@ -190,7 +200,6 @@
      VERSION "${BROTLI_ABI_COMPATIBILITY}.${BROTLI_ABI_AGE}.${BROTLI_ABI_REVISION}"
      SOVERSION "${BROTLI_ABI_COMPATIBILITY}")
    if(NOT BROTLI_EMSCRIPTEN)
 -    set_target_properties(${lib} PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
    endif()
    set_property(TARGET ${lib} APPEND PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${BROTLI_INCLUDE_DIRS}")
  endforeach()
-@@ -217,15 +226,13 @@ endif()
+@@ -217,15 +226,12 @@
  # Build the brotli executable
  add_executable(brotli ${BROTLI_CLI_C})
  target_link_libraries(brotli ${BROTLI_LIBRARIES_STATIC})
 +set_target_properties(brotli PROPERTIES EXCLUDE_FROM_ALL ON EXCLUDE_FROM_DEFAULT ON)
  
  # Installation
- if(NOT BROTLI_EMSCRIPTEN)
+-if(NOT BROTLI_EMSCRIPTEN)
```

Comment:
Have you submitted the fix upstream? I mean, why brotli didn't create an install target for emscripten in the first place?